public/content/lessons/2016/cross-products/index.mdx

```diff
@@ -1,10 +1,253 @@
 ---
 title: Cross products
-description: The cross product is a way to multiple to vectors in 3d.  This video shows how to visualize what it means.
+description: The cross product is a way to multiple to vectors in 3d. This video shows how to visualize what it means.
 date: 2016-09-01
 chapter: 10
 video: eu6i7WJeinw
 source: _2016/eola/chapter8p2.py
 credits:
 - Lesson by Grant Sanderson
+- Text adaptation by Kurt Bruns
 ---
+
+> "Every dimension is special."
+>
+> \- Jeff Lagarias
+
+Last chapter, I talked about the dot product, showing both the standard introduction to the topic, as well as the deeper view of it related to linear transformations. I'd like to do the same for cross products, which also have a standard introduction along with a deeper understanding in the light of linear transformations, but this time I'm dividing it into two chapters.
+
+Here, I'll try to hit the main points that students are usually shown about the cross product, and in the next chapter I'll show a view which is less commonly taught, but really satisfying when you learn it.
+
+## Two dimensions
+
+We'll start in two dimensions. If you have two vectors, $\vec{\mathbf{v}}$ and $\vec{\mathbf{w}}$, think about the parallelogram that they span out. What I mean by this is that if you take a copy of $\vec{\mathbf{v}}$, and move its tail to the tip of w, and you take a copy of $\vec{\mathbf{w}}$, and move its tail to the tip of $\vec{\mathbf{v}}$, the four vectors enclose a certain parallelogram.
+
+<Figure
+    image="./figures/two-dimensions/AreaOfParallelogram.svg" 
+/>
+
+The cross product of $\vec{\mathbf{v}}$ and $\vec{\mathbf{w}}$, written with the x-shaped multiplication symbol ($\times$), is the area of this parallelogram.
+
+<Figure
+    image="./figures/two-dimensions/AreaOfParallelogramFormula.svg" 
+/>
+
+Well, almost, we also need to consider orientation. Basically, if $\vec{\mathbf{v}}$ is on the right of $\vec{\mathbf{w}}$, then $\vec{\mathbf{v}}$ cross $\vec{\mathbf{w}}$ is positive and equal to the area of that parallelogram.
```

Comment:
```suggestion
Well, almost, we also need to consider orientation.  If $\vec{\mathbf{w}}$ is a counterclockwise rotation away from $\vec{\mathbf{w}}$, then $\vec{\mathbf{v}} \times \vec{\mathbf{w}}$ is positive and equal to the area of that parallelogram.
```

Reply:
I realize the phrase "to the right of" is a little confusing when the vectors are upside down. It's also not exactly how I think about it in my own head (I'm curious how you do). Instead what's relevant is which direction one vector would need to rotate to get to the other, when opting for the shorter rotation.

Maybe "is a counterclockwise rotation away from" is also clunky, I'm open to suggestions!

Reply:
I agree that "to the right of" is a bit confusing. I typically think in terms of "clockwise rotation" and "counterclockwise rotation", which is what I'll change it to for now.

However, what do you think of the terminology: 

- "If $\vec{\mathbf{v}}$ comes before $\vec{\mathbf{w}}$ in a clockwise direction, then..."
- "If $\vec{\mathbf{v}}$ comes after $\vec{\mathbf{w}}$ in a clockwise direction, then..."

I think a student would be right to object that both are true, but maybe using *before* and *after* encodes some sense of the smaller rotational arc indicating which one we mean.

Reply:
After changing it to "counterclockwise" and "clockwise" I think it reads pretty well.